### DISCUSSION TITLE
Is there any HTTP handler ?

### DISCUSSION THREAD

**Discussion by krishnaTORQUE** [Rahix/avr-hal#327] [Q&A] [answered]
Is there any HTTP handler ?
Sorry I did not find any, if yes then can anyone please send me a link?

**Comment by stappersg** [comment]
This issue should be closed to prevent further drain of human energy.

@krishnaTORQUE   please read and **understand** http://www.catb.org/~esr/faqs/smart-questions.html

Then start search for a _TCP/IP stack_ for AVR micro-controllers, when found you have something to stack _HTTP_ on top.
Meanwhile wonder about **your choice** for  the AVR and HTTP combination.

**Comment by Rahix** [comment, marked as answer]
As @stappersg has essentially said already: HTTP requires an entire networking stack which is 1. completely out of scope for `avr-hal` and 2. quite a beast to get working.  I have not seen anyone working on this so far.  If you really need it, you could try linking in a C library which does what you need but this is also not trivial...

So I'll close this issue as there is unfortunately nothing in `avr-hal` which we can do to help with this...

**Comment by djdisodo** [comment]
may [this](https://github.com/smoltcp-rs/smoltcp) help?

handling http is relativly easy
most of the web browser will just work even if you don't implement the whole
